usage: test.py [-h] [--run_idx RUN_IDX]
test.py: error: unrecognized arguments: --model_idx 2
